services/harmony/app/util/edl-api.ts

```diff
@@ -215,3 +215,21 @@ export async function validateUserIsInCoreGroup(
 
   return true;
 }
+
+/**
+ * Returns true if EDL is up
+ *
+ * @param context - request context
+ * @returns A Promise containing `true` if EDL is up
+ */
+export async function isEdlHealthy(context: RequestContext): Promise<boolean> {
+  const { logger } = context;
+  try {
+    const response = await axios.default.get(`${env.oauthHost}/home`);
```

Comment:
nit: you are using `fetch` in `isCmrHealthy`, but `axios` here. Any reason for not using the same in both?

Reply:
Because CMR namespace is using `fetch` and edl-api namespace is using `axios`. I am just trying to be consistent within the same namespace. As to why these namesapces use different libraries, it is beyond me.